lists-service/cicd/deploy/pipeline/deploy_lists_buildspec.yaml

```diff
@@ -0,0 +1,50 @@
+version: 0.2
+
+env:
+  shell: bash
+  variables:
+    JAVA_TOOL_OPTIONS: -Dhttps.protocols=TLSv1.2
+  secrets-manager:
+    DB_PASSWORD: $LISTS_SECRET_NAME:db-password
+
+phases:
+  install:
+    commands:
+      - echo Installing dependencies...
+      - apt update -y
+        && cat /etc/lsb-release
+        && apt-get -q -y install openjdk-11-jdk
+        && apt-get -q -y install unzip
+        && apt-get -q -y install zip
+        && apt-get -q -y install curl
+        && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+        && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+        && apt-get update
+        && apt-get install -y docker-ce
+        && curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+        && chmod 700 get_helm.sh
+        && ./get_helm.sh
+        && curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+        && chmod +x kubectl
+        && mv kubectl /usr/local/bin/
+        && ARCH=amd64
+        && PLATFORM=$(uname -s)_$ARCH
+        && curl -sLO "https://github.com/eksctl-io/eksctl/releases/latest/download/eksctl_$PLATFORM.tar.gz"
+        && tar -xzf eksctl_$PLATFORM.tar.gz -C /tmp && rm eksctl_$PLATFORM.tar.gz
+        && mv /tmp/eksctl /usr/local/bin
```

Comment:
Most of these dependencies are already present in the build container, removing them will make deploys faster and simplify this. [Userdetails](https://github.com/AtlasOfLivingAustralia/userdetails/blob/feature/containerization/cicd/deploy/pipeline/deploy_userdetails_buildspec.yaml#L10-L12)